numpy/lib/nanfunctions.py

```diff
@@ -548,7 +551,7 @@ def nanprod(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
     Parameters
     ----------
     a : array_like
-        Array containing numbers whose sum is desired. If `a` is not an
+        Array containing numbers whose product is desired. If `a` is not an
         array, a conversion is attempted.
```

Comment:
My mistake. By the way, would it make sense to apply the `_ureduce` function from `numpy.lib.function_base` to other places, like most of `numpy.core.from_numeric`? That way we could add multi-axis support without waiting for the `gufunc` redesign that seems to be coming.